src/lib/common/tag.h

```diff
@@ -30,6 +30,22 @@
 #include "common/Format.h"
 
 
+// Macros to be used for JSON rendering
+
+#define JSON_PROP(name) std::string("\"" + std::string(name) + "\":")
```

Comment:
Of course. The '+' operator of std::string accepts char* as input (among other types).
That's why I started with std::string in my example (I have suffered it too ... believe me).

I wonder how 

    "\"" + std::string(name)

works ...

This is 'char*' + 'std::string' ... Did you compile this?